### ISSUE TITLE
paper-input temporarily creates a horizontal scrollbar after typing first letter

### ISSUE THREAD

**Issue by johnbradley** [PolymerElements/paper-input#277]
This can be seen in the demo.
https://elements.polymer-project.org/elements/paper-input?view=demo:demo/index.html&active=paper-input
Just type a letter in one of the paper-input textboxes where the label is setup to float upwards.
While the label is floating upwards there is a horizontal scrollbar.
It only occurs when the first character is typed since that is when the label floats upward.

It looks like it is related to the "width: 133%;" specified in paper-input-container.html.
```
      .input-content.label-is-floating ::content label,
      .input-content.label-is-floating ::content .paper-input-label {
      ....
          /* Since we scale to 75/100 of the size, we actually have 100/75 of the
          original space now available */
          width: 133%; 
```
Changing this to width:100% seems to fix the issue, but probably messes up whatever is going on with the scaling.

**Comment by Gwalks91** [comment]
+1

**Comment by MeTaNoV** [comment]
I have the same strange behavior in one of my demo

**Comment by robrez** [comment]
I worked around it like this:
```
      paper-input,
      paper-textarea {
        position: relative;
        overflow: hidden;
      }
```

**Comment by notwaldorf** [comment]
Fixed in #294
